src/task/ReceiveVideoInputTask.ts

```diff
@@ -120,24 +120,13 @@ export default class ReceiveVideoInputTask extends BaseTask {
       );
       const trackSettings = videoTracks[0].getSettings();
 
-      if (this.context.enableSimulcast) {
-        // For video, we currently let uplink policy to specify constraints for simulcast. This logic should be removed in the future.
```

Comment:
I checked with Shi and he agreed this divergence wasn't necessary. I think we were just playing it safe, initially.